spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/repository/query/SqlSpannerQuery.java

```diff
@@ -245,8 +245,7 @@ private List executeReadSql(Pageable pageable, Sort sort, QueryTagValue queryTag
 
 		queryTagValue.sql = SpannerStatementQueryExecutor
 				.applySortingPagingQueryOptions(this.entityType, spannerQueryOptions,
-						resolveEntityClassNames(queryTagValue.sql),
```

Comment:
could you clarify why is the call to `resolveEntityClassNames` removed?

Reply:
Ok so `executeReadSql` is only called from `executeRawResult`, and `resolveEntityClassNames` is done there. 

I think the better way to fix that would be to call `resolveEntityClassNames` in the constructor, this way the `sql` field will always have resolved class names and there will be no need for any individual method to do that.

Reply:
it will be added to the PR